convert2rhel/actions/system_checks/package_updates.py

```diff
@@ -68,12 +100,25 @@ def run(self):
                 if not reposdir
                 else "on repositories defined in the %s folder" % reposdir
             )
-            logger.warning(
+            package_not_up_to_date_error_message = (
                 "The system has %s package(s) not updated based %s.\n"
                 "List of packages to update: %s.\n\n"
                 "Not updating the packages may cause the conversion to fail.\n"
                 "Consider updating the packages before proceeding with the conversion."
                 % (len(packages_to_update), repos_message, " ".join(packages_to_update))
             )
+            logger.warning(package_not_up_to_date_error_message)
+            self.set_result(
+                level="SUCCESS",
+                id="OUT_OF_DATE_PACKAGES",
+                title="Outdated packages detected",
+            )
+            self.add_message(
+                level="WARNING",
+                id="PACKAGE_NOT_UP_TO_DATE_MESSAGE",
+                title="Outdated packages detected",
+                description="Please refer to the diagnosis for further information",
+                diagnosis=package_not_up_to_date_error_message,
```

Comment:
We have many of these where we say look at diagnosis inside the description. Seems a bit odd to me

Reply:
For most of those it is a case where the valuable information is found in the diagnosis, sometimes it is an exception raised which we don't know the contents of - so the description just points to the user to the valuable info in the diagnosis. Additionally, we can only store text with variables in the diagnosis and remediation fields currently since they will be templated, which is why were not just adding the diagnosis info into the description field